inst/app/global.R

```diff
@@ -11,8 +11,8 @@
 ## data wrangling
 library(dplyr)
 library(tidyr)
-library(googlesheets4)
 library(sf)
+library(hkdatasets) # Installed with tarball. See https://github.com/Hong-Kong-Districts-Info/hkdatasets
```

Comment:
No need for the comment because it's automatically downloaded from the `renv.lock` file.
```suggestion
library(hkdatasets)
```